index.bs

```diff
@@ -59,6 +59,8 @@ Every {{Document}} has a <dfn for="Document">prerendering browsing contexts map<
 
 <p class="issue">Should this map be scoped to the [=user agent=] instead? Or, allowed to be copied between documents?
 
+Every {{Document}} has a <dfn for="Document">post-activation steps list</dfn>, which is a [=list=] where each [=list/item=] is a series of algorithm steps.
```

Comment:
I suspect at this granularity we'd want to use a more verbose term than "activation", since the term "active" is already used for other purposes (e.g., `document.activeElement`).